styles/styles.css

```diff
@@ -0,0 +1,264 @@
+body {
+    font-family: 'Berkshire Swash', cursive;
+}
+p, nav {
+    font-family: 'Anaheim', sans-serif;
+}
+/*Header*/
+.header {
+    background-image: url(/images/promo-background.jpeg);
+    height: 420px;
+    width: 99.5%vw; 
+    color: white;
+}
+.top-container {
+    display: flex;
+}
+/*Logo*/
+.logo {
+    margin-top: 10px;
+    margin-left: 40px;
+    margin-right: -60px;
+}
+/*End of logo*/
+ul {
+    list-style-type: none;
+    display: inline-flex;
+    padding: 20px;
+    color: white;
+    font-family: 'Berkshire Swash', cursive;;
+    font-size: small;
+    margin-left: 240px;
+}
+ul a {
+    padding: 10px;
+    color: white;
+}
+ul a:hover {
+    background-color: darkgreen;
+    opacity: 30%;
+    color: white;
+}
+ul a.current {
+    background-color: darkred;
+    opacity: 35%;
+}
+ul a.current:hover {
+    opacity: 60%;
+}
+.header > h1 {
+    text-align: center;
+    font-size: 50px;
+
+}
+.header > p {
+    text-align: center;
+}
+.buttons {
+    text-align: center;
+}
+.btn-1 {
+    background-color: black;
+    height: 35px;
+    width: 140px;
+    opacity: 60%;
+    color: white;
+    border: none;
+}
+.btn-1:hover {
+    opacity: 100%;
+    border: white 1px solid;
+}
+.btn-2 {
+    background-color: white;
+    height: 35px;
+    width: 140px;
+    opacity: 30%;
+    color: black;
+    border: none;
+}
+.btn-2:hover {
+    opacity: 100%;
+    border: darkgrey 1px solid;
+}
+/*Cook and dish (Just the right food)*/
+.cook {
+    background-image: url(/images/about-background.jpeg);
+    height: 400px;
+    width: 99.5% vw;
```

Comment:
Eeek! Looks like you're mixing two units, `%` and `vw`. They are not the same, though the operate similarly. `%` takes a percentage of the parent container, and `vw` takes a percentage of the viewport width (which is the viewable area in the browser). In this case the browser will probably treat this as `%`